inc/vcf/error.hpp

```diff
@@ -269,8 +269,15 @@ namespace ebi
         using BodySectionError::BodySectionError;
```

Comment:
`BodySectionError` could contain a new attribute `num_column` to avoid using magic numbers in the fixer `visit` methods.